### PR TITLE
fix get hv_vmbus version issue, the raw output contains "\r\n"

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/tools/modinfo.py [hunk 1]
```diff
@@ -59,7 +59,11 @@ def get_version(
             no_info_log=True,
             no_error_log=True,
         )
-        return get_matched_str(output, self._version_pattern)
+        for line in output.splitlines():
```
Comment (squirrelsc) [inline lisa/tools/modinfo.py]: Can it be fixed by add "re.M" on the pattern?

File: lisa/tools/modinfo.py [hunk 2]
```diff
@@ -59,7 +59,11 @@ def get_version(
...
+            version = get_matched_str(line, self._version_pattern)
+            if len(version) > 0:
+                return version
+        return ""
```
Comment (dsrivastavv) [inline lisa/tools/modinfo.py]: If no version is found, should it throw an error? or is this behaviour normal for some distros?

Comment (LiliDeng) [reply lisa/tools/modinfo.py]: some modules don't have version

**Review by squirrelsc — APPROVED** [review]

File: lisa/tools/modinfo.py (hunk 1 above)
Comment (LiliDeng) [reply lisa/tools/modinfo.py]: yes, fixed by adding re.M, updated and tested

**Pull request merged**